pkgs/development/cuda-modules/flags.nix

```diff
@@ -207,6 +211,16 @@ let
       # E.g. "-gencode=arch=compute_75,code=sm_75 ... -gencode=arch=compute_86,code=compute_86"
       gencodeString = strings.concatStringsSep " " gencode;
 
+      # cmakeCudaArchitectures :: List String
+      # A list of CUDA capabilities without dots, suitable for passing to CMake.
+      # E.g. [ "75" "86" ]
+      cmakeCudaArchitectures = lists.map dropDot cudaCapabilities;
```

Comment:
A shorthand for cmake sounds reasonable, but let's omit this one since it's so easily derivable from `cudaFlags` and probably not directly consumed anywhere else?

Reply:
I did it mostly for the hassle or avoiding the need to re-write `dropDot` in multiple places. It's my preference to keep it, but if you're strongly opposed to it I can remove it. Thoughts @SomeoneSerge?

Reply:
I don't see any other usages of `cmakeCudaArchitectures` 🤔

Reply:
Resolved in af1ffadd2d8a00fe1edf8f57d0affd04fc5d95a9